### PR TITLE
Remove internal chart generator

### PR DESCRIPTION
**Pull request by ebadyano**

### CODE REVIEW

**Comment by ebadyano** [comment]
@elasticmachine run rally/it-python38

**Review by pquentin — CHANGES_REQUESTED** [review]
The failure seems related:

> 22:59:41 esrally: error: argument subcommand: invalid choice: 'generate' (choose from 'race', 'list', 'delete', 'info', 'create-track', 'compare', 'build', 'download', 'install', 'start', 'stop')

**Comment by ebadyano** [comment]
> The failure seems related:
> 
> > 22:59:41 esrally: error: argument subcommand: invalid choice: 'generate' (choose from 'race', 'list', 'delete', 'info', 'create-track', 'compare', 'build', 'download', 'install', 'start', 'stop')

missed removing the generate test.

File: it/generate_test.py [hunk 1]
```diff
@@ -1,35 +0,0 @@
...
-
-import os
-
-import it
-
-
-@it.rally_in_mem
-def test_track_info_with_challenge(cfg, tmp_path):
-    cwd = os.path.dirname(__file__)
-    chart_spec_path = os.path.join(cwd, "resources", "sample-race-config.json")
```
Comment (pquentin) [inline it/generate_test.py:26]: Can you please remove this file from the resources directory?

**Review by pquentin — APPROVED** [review]
Thanks! This looks great. Left two nits. We don't need another round of review here.

Can you please replace "avoid breaking the chart generator" with "avoid breaking the charts" in esrally/driver/driver.py?

File: it/generate_test.py (hunk 1 above)
Comment (ebadyano) [reply it/generate_test.py:26]: good catch thanks!

**Pull request merged**